configure.ac

```diff
@@ -1224,7 +1230,24 @@ if test x$enable_wallet != xno; then
     fi
 
     dnl Check for sqlite3
-    PKG_CHECK_MODULES([SQLITE], [sqlite3 >= 3.7.17], , [AC_MSG_ERROR([sqlite3 not found.])])
+    if test "x$use_sqlite" != "xno"; then
+      PKG_CHECK_MODULES([SQLITE], [sqlite3 >= 3.7.17], [have_sqlite=yes], [have_sqlite=no])
+    fi
+    AC_MSG_CHECKING([whether to build wallet with support for sqlite])
+    if test "x$use_sqlite" = "xno"; then
+      use_sqlite=no
```

Comment:
5c19bd0f4b893a91c02e652ac8f45ef0ccdeca1c

is this a tautology?

Reply:
Yes, but not having anything here would be a bug.